packages/ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@prismarine-web-client/ui",
+  "version": "1.0.0",
+  "main": "./src/index.ts",
+  "scripts": {
```

Comment:
Add `private: true` for now we're not going to publish it since I don't own @prismarine-web-client